src/index.d.ts

```diff
@@ -273,9 +273,24 @@ declare module 'discord-akairo' {
 
         public addMessage(message: Message | Message[]): Message | Message[];
         public edit(content?: StringResolvable, options?: MessageEmbed | MessageEditOptions): Promise<Message>;
```

Comment:
You should separate edit out too, as with reply and send, for content and options.